<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>1034</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_threshold">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_mAP_prototxt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mAP_prototxt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mAP_prototxt:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mean_value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>prototxt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_model">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>421</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>/home/onelly/model/1109_withbias_bnmerge.caffemodel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prototxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>421</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>/home/onelly/model/test_bnmerge.prototxt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_model">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>model:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mean_value">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>104,117,123</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mAP2_prototxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>421</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>/home/onelly/model/test_bnmerge.prototxt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>threshold:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_prototxt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>prototxt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>init</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>quntize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Net</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>151</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>origin:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>layer detail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>to change:</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>151</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>191</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>layer detail:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>620</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start And Save As ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>map</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>571</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>291</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>id:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>name:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>291</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>151</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>190</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>quantize list:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>870</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU tmpr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gname">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>840</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>900</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gtmpr">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>870</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>900</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU mem</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_guse">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>930</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>870</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>840</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>930</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>890</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>GPU Infomation</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gmem">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>900</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>930</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU use</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
